packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -2244,58 +2244,3 @@ Generated by [AVA](https://avajs.dev).
     ────────────────────────────────────────────────────────────────␊
     ␊
     (Netlify Build completed in 1ms)`
-
-## Generates a `manifest.json` file when running outside of buildbot
```

Comment:
I've added `snapshot: false` to the test, since we just need to verify that the file has been created on disk.